metrotranscom/doorway
# fix: enabling common app functionality

# Pull Request Template

## Issue Overview

This PR addresses #502 


## Description

This PR enables the common application selection on the partners side and the logic to show the apply online button and/or the helper text below.

## How Can This Be Tested/Reviewed?

This can be tested by turning the common app on or off for an existing listing. I had more difficulty testing the helper text since even after running "yarn db:reseed:with-external:prod", I would see the external listings from the listings page but got page not found errors when trying to see the details page where the apply online would be. One way to test this is by manually flipping the prop on line 381 of ListingView.tsx. 

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
